domain/schema/model/sql/0017-machine.sql

```diff
@@ -73,3 +73,9 @@ CREATE TABLE machine_filesystem (
     REFERENCES storage_filesystem (uuid),
     PRIMARY KEY (machine_uuid, filesystem_uuid)
 );
+
+CREATE TABLE machine_requires_reboot
+(
```

Comment:
Can we get the dangling `(` on the same line as `CREATE TABLE ...` so that it's consistent with the rest of the file.

Reply:
Oops, should've look twice. I am lazy and thought that sqlfluff would do that for me ^^